src/github.ts

```diff
@@ -337,6 +337,7 @@ export class GitHub {
           }
           commits.push([commit.sha, commit.commit.message]);
         }
+        found = response.data.length < per_page || found;
```

Comment:
I'm not 100% sure that we wouldn't sometimes get smaller page sizes when we haven't iterated over the whole dataset. Perhaps alternatively we could pick a reasonable upper-bound on the response size?

N pages back in time?

Reply:
@abraghib I'd like to test the `response.data.length === 0` idea, but it seems like a reasonable approach.